docs/DeveloperGuide.md

```diff
@@ -286,7 +334,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`    | long-time user                            | separate past and current clients                                                    | avoid contacting clients with similar names or old clients that I am not presently working with   |
 | `* *`    | new user                                  | view a list of commands                                                              | know what commands are available and the right commands to use                                    |
 | `*`      | interior designer                         | add images under a client                                                            | keep track of images such as floor plans and inspirational designs relevant to the client         |
-| `*`      | interior designer                         | add a high importance tag to a client                                                | take note of pressing issues regarding a client, such as mobility issues                          |
```

Comment:
Since we're adding this, it might be better to up the priority of the user story!

Reply:
Thanks for the reminder! 👍 I'll push the changes asap